etc/nsjail/execute.cfg

```diff
@@ -0,0 +1,137 @@
+name: "compiler explorer compilation/tool sandbox"
+
+mode: ONCE
+hostname: "ce"
+
+time_limit: 0
+max_cpus: 1
+
+log_level: INFO
+
+rlimit_as_type: INF
+rlimit_cpu_type: SOFT
+rlimit_fsize: 1024 # 1GB
+rlimit_nofile: 20
+
+uidmap {
+    inside_id: "0"
+}
+
+gidmap {
+    inside_id: "0"
+}
+
+# must run following as root during system startup
+# cgcreate -a ubuntu:ubuntu -g memory,pids,cpu,net_cls:ce-compile
+cgroup_mem_parent: "ce-compile"
+cgroup_pids_parent: "ce-compile"
+cgroup_net_cls_parent: "ce-compile"
+cgroup_cpu_parent: "ce-compile"
+
+cgroup_mem_max: 1342177280 # 1.25 GiB
+cgroup_pids_max: 4
+cgroup_cpu_ms_per_sec: 800
```

Comment:
this should be set to `1000`

This will let the process group use a single core at 100%, or two cores at 50%

contrasting with max_cpus above, which uses `sched_setaffinity` to pick a random cpu to bind the group to, which can be sub-optimal if multiple compilations or sandbox executions end up picking the same core at once.

Upon further review, for this reason, we probably want to remove the max_cpus from our sandbox config too and rely purely on `cgroup_cpu_ms_per_sec`.

Reply:
> Upon further review, for this reason, we probably want to remove the max_cpus from our sandbox config too and rely purely on cgroup_cpu_ms_per_sec.

Will do.